README.md

```diff
@@ -55,6 +55,22 @@ SSL is enabled by default. You can disable it by passing a empty option list `[]
 TPusherServer.Create('app_id', 'app_key', 'secret', '', []);
 ```
 
+### Async
+
+To use async request you should use the `TAsyncPusherServer` class:
+```
+AsyncPusherServer := TAsyncPusherServer.Create('app_id', 'app_key', 'secret');
```

Comment:
Makes sense, but, I don't know if it is a good idea.

Indeed, it probably doesn't make sense to use this class without this event, but I'm not sure it is worth to add it to the constructor. 

I don't like the syntax for passing an anonymous method to a constructor which already have other parameters.